zhmcclient/_client.py

```diff
@@ -61,6 +63,15 @@ def cpcs(self):
         """
         return self._cpcs
 
+    @property
+    def metrics_contexts(self):
+        """
+        :class:`~zhmcclient.MetricsContextManager`:
+          Manager object for the MetrcsContexts in scope of this client
```

Comment:
* Typo "MetrcsContexts"
* In the text, we should either talk about "Metrics Contexts" where both words are a link to the same-named glossary term, or about "MetricsContext objects" where the first word is a link to the same-named class.